### DISCUSSION TITLE
Missing issue #312 on the "normal quality doesn't exist" error

### DISCUSSION THREAD

**Discussion by HackUnit** [pystardust/ani-cli#329] [Q&A] [answered]
I am having this issue, but cannot find issue #312 to give input or find a solution? I'm still trying to figure out git and everything to actually be able to contribute, but I know it's my fault I can't find this. Can anyone point me in the right direction? I'm sorry if I'm double posting something someone else has. Tried searching, but came up null.

Thank you.

**Comment by port19x** [comment, marked as answer]
Funny, if you read back your discussion the #312 should hyperlink to the issue.
It's depreceated now anyways, #217 could use your attention.
If you are a windows or mac user, working on better support for those operating systems is more important
